tests/model_registry/rbac/test_mr_rbac.py

```diff
@@ -0,0 +1,271 @@
+            test_idp_user_session: The test user session containing both admin and user contexts
+            model_registry_group_with_user: The Model Registry group with the test user added
+
+        Raises:
+            AssertionError: If access permissions don't match expectations
+            ForbiddenException: Expected before group addition, unexpected after
+        """
+        LOGGER.info(
+            "-----Test that a user can access to the Model Registry once added to a "
+            "group that has the permissions to access it-----"
+        )
+        # Verify group membership
+        verify_group_membership(
+            group=model_registry_group_with_user,
+            username=test_idp_user_session.username,
+        )
+
+        # Wait for access to be granted
+        with switch_user_context(test_idp_user_session.user_context):
+            sampler = TimeoutSampler(
+                wait_timeout=240,
+                sleep=5,
+                func=assert_positive_mr_registry,
+                model_registry_instance=model_registry_instance,
+                model_registry_namespace=model_registry_namespace,
+                admin_client=admin_client,
+            )
+            for _ in sampler:
+                break  # Break after first successful iteration
+            LOGGER.info("Successfully accessed Model Registry")
+
+    @pytest.mark.sanity
+    @pytest.mark.parametrize(
+        "new_group",
+        [
+            pytest.param(
+                NEW_GROUP_NAME,
+                id="new_group",
+            ),
+        ],
+        indirect=["new_group"],
+    )
+    @pytest.mark.usefixtures("mr_access_role", "new_group")
+    def test_create_group(
+        self: Self,
+        model_registry_instance: ModelRegistry,
+        model_registry_namespace: str,
+        admin_client: DynamicClient,
+        test_idp_user_session: UserTestSession,
+        mr_access_role: Role,
+    ):
+        """
+        Test creating a new group and granting it Model Registry access.
+
+        This test verifies that:
+        1. A new group can be created
+        2. The group can be granted Model Registry access via RoleBinding
+        3. Users in the group can access the Model Registry
+
+        Args:
+            model_registry_instance: The Model Registry instance to test against
+            model_registry_namespace: The namespace where Model Registry is deployed
+            admin_client: The admin client for accessing the cluster
+            test_idp_user_session: The test user session containing both admin and user contexts
+            mr_access_role: The Role that grants Model Registry access
+
+        Raises:
+            AssertionError: If group creation or access permissions don't match expectations
+        """
+        LOGGER.info(
+            "-----Test that a new group can be granted access to Model Registry and user added to it can access MR-----"
```

Comment:
Can you please remove the leading and trailing "----" from all the log messages?